bitnami/wordpress/templates/hpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2beta2
```

Comment:
It would be great to still support older k8s versions. To achieve this we have some helpers in the `bitnami/common` chart to set the `apiVersion` depending on which k8s version is used. See: https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl

A new helper for the autoscaling version should be added to the `bitnami/common` chart that could then be used here. Would you mind opening a PR for this?